std/algebra/emulated/sw_bn254/pairing.go

```diff
@@ -120,21 +123,30 @@ func (pr Pairing) FinalExponentiationTorus(e *GTEl) *fields_bn254.E6 {
 	t2 = pr.InverseTorus(c)
 	t2 = pr.MulTorus(t2, t3)
 	t2 = pr.FrobeniusCubeTorus(t2)
-	t0 = pr.MulTorus(t2, t0)
 
-	return t0
+	// MulTorus(t0, t2) requires t0 ≠ t2. When this is the case it means the
```

Comment:
right! the comment is incorrect. It should be `t0 ≠ -t2` to avoid division by 0 in `MulTorus()` the sum:
https://github.com/ConsenSys/gnark/blob/17e2743737b581311a00064ed559d67eea859762/std/algebra/emulated/fields_bn254/e12_pairing.go#L229